src/organisms/cards/SimplePolicyCard/policy_card.module.scss

```diff
@@ -172,9 +177,8 @@ $footer: #f6f6f6;
   }
 
   .compare {
-    @include section(0, 17.5%, true);
     label { margin-left: rem-calc(8px); }
-    padding: 0;
+    padding: 0 ru(1.25);
```

Comment:
Haven't rhythm units fallen out of favor?

Reply:
👍